src/components/comment/CommentVoteForm.tsx

```diff
@@ -1,72 +1,75 @@
 'use client';
+import React, { useCallback, memo } from 'react';
 import { voteHandlerAction } from '@/actions/commentVote';
 import { useAction } from '@/hooks/useAction';
 import { VoteType } from '@prisma/client';
-import { ThumbsDownIcon, ThumbsUpIcon } from 'lucide-react';
+import ThumbUpOffAltOutlinedIcon from '@mui/icons-material/ThumbUpOffAltOutlined';
+import ThumbDownAltOutlinedIcon from '@mui/icons-material/ThumbDownAltOutlined';
+import ThumbUpIcon from '@mui/icons-material/ThumbUp';
+import ThumbDownIcon from '@mui/icons-material/ThumbDown';
 import { usePathname } from 'next/navigation';
-import React from 'react';
 import { toast } from 'sonner';
 
-const CommentVoteForm = ({
-  upVotes,
-  downVotes,
-  commentId,
-}: {
-  upVotes: number;
-  downVotes: number;
-  commentId: number;
-}) => {
-  const currentPath = usePathname();
-  const { execute } = useAction(voteHandlerAction, {
-    onSuccess: () => {
-      toast('Comment added');
-    },
-    onError: (error) => {
-      toast.error(error);
-    },
-  });
-  const handleUpvote = (e: React.FormEvent<HTMLFormElement>) => {
-    e.preventDefault();
-
-    execute({
-      commentId,
-      voteType: VoteType.UPVOTE,
-      currentPath,
+const CommentVoteForm = memo(
```

Comment:
i think here also memo is not needed as the props does not change based on any state

Reply:
yes, you are correct